### PR TITLE
Add command line options to start_fabric.sh

### PR DESCRIPTION
**Pull request by Ram-srini**
Signed-off-by: Ramakrishna Srinivasamurthy <ramakrishna.srinivasamurthy@intel.com>

### CODE REVIEW

File: scripts/start_fabric.sh [hunk 1]
```diff
@@ -60,24 +65,69 @@ check_if_cc_artifacts_exists()
...
 }
 
-if [ -f "$MINIFAB" ]; then
-    echo "minifab is already installed in $MINIFAB"
+while getopts "w:udch" OPTCHAR ; do
+    case $OPTCHAR in
+        u )
+            START_FABRIC=1
+            if [[ $STOP_FABRIC == 1 || $CLEAN_UP_WORK_DIR == 1 ]]; then
+                echo "Either one of option should be spefied [u|d|c]"
```
Comment (danintel) [inline scripts/start_fabric.sh:73]: Change to:
`"Only one of these options should be specified: [u|d|c]"`

File: scripts/start_fabric.sh [hunk 2]
```diff
@@ -60,24 +65,69 @@ check_if_cc_artifacts_exists()
...
+                exit
+            fi
+            ;;
+        d )
+            STOP_FABRIC=1
+            if [[ $START_FABRIC == 1 || $CLEAN_UP_WORK_DIR == 1 ]]; then
+                echo "Either one of option should be spefied [u|d|c]"
```
Comment (danintel) [inline scripts/start_fabric.sh:80]: See above

File: scripts/start_fabric.sh [hunk 3]
```diff
@@ -60,24 +65,69 @@ check_if_cc_artifacts_exists()
...
+                exit
+            fi
+            ;;
+        c )
+            CLEAN_UP_WORK_DIR=1
+            if [[ $START_FABRIC == 1 || $START_FABRIC == 1 ]]; then
+                echo "Either one of option should be spefied [u|d|c]"
```
Comment (danintel) [inline scripts/start_fabric.sh:87]: See above

Comment (Ram-srini) [reply scripts/start_fabric.sh]: Done

File: scripts/start_fabric.sh (hunk 2 above)
Comment (Ram-srini) [reply scripts/start_fabric.sh]: Done

File: scripts/start_fabric.sh (hunk 1 above)
Comment (Ram-srini) [reply scripts/start_fabric.sh]: Done

**Pull request merged**